esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -411,8 +466,14 @@ void EsdlServiceImpl::handleServiceRequest(IEspContext &context,
     const char *mthName = mthdef.queryName();
     context.addTraceSummaryValue(LogMin, "method", mthName);
 
-    if (m_custTrxCompileFail)
-        throw MakeStringException(-1, "%s::%s disabled due to Custom Transform errors. Review tranform template in configuration.", srvdef.queryName(), mthName);
+
+    IEsdlCustomTransform *crt = m_customRequestTransformMap.getValue(mthdef.queryMethodName());
+    if (!crt)
```

Comment:
does this require all methods (or the service) to have a crt?

Reply:
No, if the method has a crt it is used.  If not, it will 1. throw an error if the service level crt had an error, or 2. use the service level crt IF there is one.